pkg/clusteragent/admission/mutate/config/config.go

```diff
@@ -172,7 +172,7 @@ func (w *Webhook) inject(pod *corev1.Pod, _ string, _ dynamic.Interface) (bool,
 
 	}
 
-	if !autoinstrumentation.ShouldInject(pod, w.wmeta) {
```

Comment:
No longer dependent on workload meta (since we don't need to create the webhook).